main.go

```diff
@@ -183,6 +185,11 @@ func main() {
 		os.Exit(1)
 	}
 
+	if vpcID == "" {
```

Comment:
This is backwards incompatible change. How do we plan to introduce this in our deployment. What if we are not able to update VPC RC for some customers (we typically call broken clusters)?

Reply:
The controller manifest will be updated to add the required parameter in the next release.

Reply:
Also, if we do rollback, the manifest will be rolled back, not just the controller image image. So. IMO this should still be safe.